.github/workflows/build-and-upload.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v4
         with:
-          go-version: ~1.19.6
+          go-version: ~1.22.7
```

Comment:
nit: Do we need to set up Go here at all? All the build steps are done in the Dockerfile.